bt_server:
    ros__parameters:
        bt_name: spot_bt
        tick_freq: 60 # Hz
        stop_on_failure: false # Whether it should stop the tree from executing if the root fails.
